test/src/main/java/org/apache/accumulo/test/rpc/Mocket.java

```diff
@@ -141,13 +143,13 @@ public MocketServerTransport(MocketTransport mocketTransport) {
     public void listen() {}
 
     @Override
-    public void close() {
-      acceptImpl().close();
+    public TTransport accept() throws TTransportException {
+      return null;
```

Comment:
Shouldn't this return `servTrans`?

Reply:
This change does seem to fix the `ThriftBehaiviorIT` errors I was getting.

Reply:
Good catch.